modules/configure-cvo-retrieve-image-signatures.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+// * updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configure-cvo-retrieve-image-signatures_{context}"]
+= Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+
+If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can configure the Cluster Version Operator (CVO) to retrieve release image signatures from an endpoint on the local OSUS Operator.
+This configuration is done by adding the URI of OSUS Operator's signature serving endpoint to the CVO's list of signature stores.
```

Comment:
I added this line based on the draft doc, but let me know if this is too much technical detail for the end user.

Reply:
I think this line is helpful, but it might be long in the same paragraph as the `This is a one-time configuration...` qualifier?

Reply:
That sounds good, I can move it to the other paragraph for now.